tests/index.html

```diff
@@ -10,6 +10,7 @@
       }
     </style>
     <script src="./polyfill.js"></script>
+    <script src="./loader.js"></script>
```

Comment:
Why do we need this? I thought we included the loader inside the assets themselves, if we do then I wouldn't expect to need to load loader again.

Reply:
This makes it generally work more similarly to how these libraries would work with the real Ember CLI loader, because otherwise they would each have their own isolated `define` and `require`